{"timestamp":"2025-09-12T05:47:35.733632","level":"info","event":"DAG bundles loaded: dags-folder, example_dags","logger":"airflow.dag_processing.bundles.manager.DagBundlesManager"}
{"timestamp":"2025-09-12T05:47:35.734537","level":"info","event":"Filling up the DagBag from /opt/airflow/dags/indicium.py","logger":"airflow.models.dagbag.DagBag"}
{"timestamp":"2025-09-12T05:47:36.170137Z","level":"info","event":"Task instance is in running state","chan":"stdout","logger":"task"}
{"timestamp":"2025-09-12T05:47:36.170336Z","level":"info","event":" Previous state of the Task instance: TaskInstanceState.QUEUED","chan":"stdout","logger":"task"}
{"timestamp":"2025-09-12T05:47:36.171106Z","level":"info","event":"Current task name:extrair_sql","chan":"stdout","logger":"task"}
{"timestamp":"2025-09-12T05:47:36.171284Z","level":"info","event":"Dag name:indicium","chan":"stdout","logger":"task"}
{"timestamp":"2025-09-12T05:47:36.182936","level":"error","event":"Failed to establish connection to Docker host unix://var/run/docker.sock: Error while fetching server API version: ('Connection aborted.', PermissionError(13, 'Permission denied'))","logger":"airflow.task.hooks.airflow.providers.docker.hooks.docker.DockerHook"}
{"timestamp":"2025-09-12T05:47:36.183219","level":"error","event":"Task failed with exception","logger":"task","error_detail":[{"exc_type":"AirflowException","exc_value":"Failed to establish connection to any given Docker hosts.","exc_notes":[],"syntax_error":null,"is_cause":false,"frames":[{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/execution_time/task_runner.py","lineno":920,"name":"run"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/execution_time/task_runner.py","lineno":1215,"name":"_execute_task"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/bases/operator.py","lineno":397,"name":"wrapper"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/docker/operators/docker.py","lineno":482,"name":"execute"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/docker/operators/docker.py","lineno":356,"name":"cli"},{"filename":"/usr/local/lib/python3.12/functools.py","lineno":998,"name":"__get__"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/docker/hooks/docker.py","lineno":161,"name":"api_client"}],"is_group":false,"exceptions":[]}]}
{"timestamp":"2025-09-12T05:47:36.223382Z","level":"info","event":"Task instance in failure state","chan":"stdout","logger":"task"}
{"timestamp":"2025-09-12T05:47:36.223703Z","level":"info","event":"Task start","chan":"stdout","logger":"task"}
{"timestamp":"2025-09-12T05:47:36.223839Z","level":"info","event":"Task:<Task(DockerOperator): extrair_sql>","chan":"stdout","logger":"task"}
{"timestamp":"2025-09-12T05:47:36.223902Z","level":"info","event":"Failure caused by Failed to establish connection to any given Docker hosts.","chan":"stdout","logger":"task"}
